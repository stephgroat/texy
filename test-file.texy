int main()
{
  File f;
  f = open("test-hello.texy", "r");
  close(f);
  return 0;
}