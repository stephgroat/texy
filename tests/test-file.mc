int main()
{
  File f;
  Word space;
  int s;
  space = calloc(1,500);
  f = open("test-file.txt", "r");
  s = read(space, 1, 500, f);
  printword(space);
  close(f);
  free(space);
  return 0;
}
