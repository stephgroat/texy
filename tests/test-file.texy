int main()
{
  File f;
  Word space;
  int s;
  space = calloc(1,500);
  f = open("test-file.txt", "r");
  s = read(space, 1, 500, f);
  printword("Characters in test-file.txt:");
  print(s);
  close(f);
  free(space);
  return 0;
}
