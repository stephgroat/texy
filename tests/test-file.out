Characters in test-file.txt:
21
